ApollosProject/apollos-apps
# Fix: content padding

## ✏️ Description
adding padding to show all content at the end of a content item. 

## 🔬 To Test
1. pull this repo
2. open a content item that has features and see all features

## 📸 Screenshots

Before change there are more items under journal prompt.
<img width="473" alt="Screenshot 2023-01-06 at 2 35 06 PM" src="https://user-images.githubusercontent.com/16566029/211095459-9533fd45-7d04-4175-98da-c04af7b9fe0d.png">

post change can see all features.
<img width="501" alt="Screenshot 2023-01-06 at 2 31 37 PM" src="https://user-images.githubusercontent.com/16566029/211095545-bc160574-8055-4da8-aa61-5f9dd932a3f7.png">
